jukebox_radio/streams/views/marker/delete_view.py

```diff
@@ -12,8 +12,16 @@ def post(self, request, **kwargs):
         Marker = apps.get_model("streams", "Marker")
 
         marker_uuid = request.POST.get("markerUuid")
-
         marker = Marker.objects.get(uuid=marker_uuid)
         marker.archive()
 
-        return self.http_response_200()
+        # needed for React Redux to update the state on the FE
+        queue_uuid = request.POST.get("queueUuid")
```

Comment:
I've never had to pass data redundantly from the front-end to the back-end before. This probably doesn't seem ideal, but it _really_ makes the code path simpler. It's a one way flow (client > server > redux) as opposed to recombining data on the frontend (client > server > {server response + front-end data} > redux).